### PR TITLE
04 articles

### PR DESCRIPTION
**Pull request by harrycs1**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <BrowserRouter>
+      <Routes>
```
Comment (ghost) [inline src/App.jsx:9]: It is not ideal that there is nothing on the route of /. I have to manually navigate to the work you want to show me. This is not a great user experience at all.

File: src/components/ArticlesList.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Loading from './Loading'
```
Comment (ghost) [inline src/components/ArticlesList.jsx:1]: This isn't being used anywhere, on a slow connection I just have a blank screen.

File: src/App.jsx [hunk 3]
```diff
@@ -1,34 +1,26 @@
...
-import viteLogo from '/vite.svg'
+import { useState, useEffect } from 'react'
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { getArticles } from '../api';
+import { ArticlesPage } from './pages/ArticlesPage';
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```
Comment (ghost) [inline src/App.jsx]: According to your plan this shouldn't be here. Due to the size of this project, this absolutely shouldn't be here.

File: src/App.jsx [hunk 4]
```diff
@@ -1,34 +1,26 @@
...
+
+  useEffect(() => {
```
Comment (ghost) [inline src/App.jsx]: Any reason the api call for articles is not in the articles component?

**Review by ghost — CHANGES_REQUESTED** [review]
Currently there is nothing rendering for / or when I open your app. While it is your choice where articles get rendered, I shouldn't have to manually navigate using the address bar to get around your app.

Looking at the plan you provided you haven't followed it. Why is the data for all articles in app and not in the component that is taking care of articles?

You currently have no loading pattern, on a slow connection the user has no feedback that anything is happening.

On a mobile screen the titles aren't squashed which is nice. Is this all you want to show to the user? A little gap above and below the article card would be ideal for some visual spacing.

There is quite a bit to address here before you move on. Please resubmit when ready.

**Review by nc-poppy-m — APPROVED** [review]
Good work implementing this feedback! Merge when ready

**Pull request merged**